VectorMessage desired_relationship
VectorMessage actual_relationship
int32 cell_id

	# VectorMessage contains:
		# float64 x
		# float64 y
		# float64 z